src/js/descript.js

```diff
@@ -135,5 +167,41 @@
         return containerName ? this._containers[containerName] : this._containers;
     };
 
+    /**
+     * Replaces string patterns in inline scripts. Patterns can be
+     * @param containerName
+     * @param scriptAttribute
+     * @param pattern
+     * @param replacement
+     */
+    Descript.prototype.replace = function(containerName, scriptAttribute, pattern, replacement) {
+        var containerScripts = this.get(containerName).toArray();
+        var script = this._find(containerScripts, scriptAttribute);
```

Comment:
Could we refactor this part of the API a bit?

It seems for the most part we get the container scripts and then call `_find` with the container name. What if _find took the name of the container instead?